convertigo:8.2.7 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
convertigo:8.2 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
convertigo:8.2.5 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
convertigo:8.2.4 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
convertigo:8.1.0 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
flink:1.18.1-scala_2.12-java17 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
flink:1.18.1-scala_2.12-java11 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
flink:1.18.1-scala_2.12 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
flink:1.18.1-java17 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
flink:1.18.1-java11 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
swipl:stable ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
swipl:9.2.5 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
swipl:9.3.6 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
swipl:9.2.4 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
swipl:9.3.2 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
rust:alpine3.20 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
rust:alpine ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
rust:1.78.0-alpine3.20 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
rust:1.78.0-alpine ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
rust:1.78-alpine3.20 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
mediawiki:stable-fpm-alpine ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
mediawiki:stable-fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
mediawiki:stable ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
mediawiki:lts-fpm-alpine ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
mediawiki:lts-fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
sl:7 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
sl:6 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
open-liberty:kernel-slim-java8-openj9 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
open-liberty:kernel-slim-java17-openj9 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
open-liberty:kernel-slim-java11-openj9 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
open-liberty:kernel-slim ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
open-liberty:full-java8-openj9 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
clefos:7.7.1908 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_clefos:7.7.1908_2024-06-13_09:07:42.json clefos:7.7.1908] execution failed: exit status 1
Output: 2024-06-13T09:07:43Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:07:44Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "clefos:7.7.1908" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (clefos:7.7.1908): Error response from daemon: No such image: clefos:7.7.1908
	* containerd error: image not found in containerd store: clefos:7.7.1908
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: no child with platform linux/amd64 in index clefos:7.7.1908


 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> Command [/usr/local/bin/grype -o sarif --file /root/docker-vulnerability-analyzer/out/official/new9/Grype_scan_clefos:7.7.1908_2024-06-13_09:07:42.json clefos:7.7.1908] execution failed: exit status 1
Output: [31mfailed to catalog: errors occurred attempting to resolve 'clefos:7.7.1908':
  - docker: unable to inspect existing image: Error response from daemon: No such image: clefos:7.7.1908
  - podman: podman not available: no host address
  - containerd: pull failed: no match for platform in manifest: not found
  - oci-registry: failed to get image from registry: no child with platform linux/amd64 in index clefos:7.7.1908
  - additionally, the following providers failed with file does not exist: docker-archive, oci-archive, oci-dir, singularity, oci-dir, local-file, local-directory[0m
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> Command [/usr/local/bin/snyk container test clefos:7.7.1908 --sarif-file-output=/root/docker-vulnerability-analyzer/out/official/new9/Snyk_scan_clefos:7.7.1908_2024-06-13_09:07:42.json] execution failed: exit status 2
Output: The image does not exist for the current platform
 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
clefos:7 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_clefos:7_2024-06-13_09:07:58.json clefos:7] execution failed: exit status 1
Output: 2024-06-13T09:07:59Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:08:00Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "clefos:7" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (clefos:7): Error response from daemon: No such image: clefos:7
	* containerd error: image not found in containerd store: clefos:7
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: no child with platform linux/amd64 in index clefos:7


 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> Command [/usr/local/bin/grype -o sarif --file /root/docker-vulnerability-analyzer/out/official/new9/Grype_scan_clefos:7_2024-06-13_09:07:58.json clefos:7] execution failed: exit status 1
Output: [31mfailed to catalog: errors occurred attempting to resolve 'clefos:7':
  - docker: unable to inspect existing image: Error response from daemon: No such image: clefos:7
  - podman: podman not available: no host address
  - containerd: pull failed: no match for platform in manifest: not found
  - oci-registry: failed to get image from registry: no child with platform linux/amd64 in index clefos:7
  - additionally, the following providers failed with file does not exist: docker-archive, oci-archive, oci-dir, singularity, oci-dir, local-file, local-directory[0m
 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> Command [/usr/local/bin/snyk container test clefos:7 --sarif-file-output=/root/docker-vulnerability-analyzer/out/official/new9/Snyk_scan_clefos:7_2024-06-13_09:07:58.json] execution failed: exit status 2
Output: The image does not exist for the current platform

teamspeak:3.13.7 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
teamspeak:3.13 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
teamspeak:3.13.6 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
teamspeak:3.13.5 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
teamspeak:3.13.3 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
matomo:fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
matomo:apache ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
matomo:5.0.3-fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
matomo:5.0.3-apache ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
matomo:5.0.3 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
yourls:fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
yourls:apache ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_yourls:apache_2024-06-13_09:12:58.json yourls:apache] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image yourls:apache: failed to pull image yourls:apache: GET https://index.docker.io/v2/library/yourls/manifests/apache: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
yourls:1.9.2-fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_yourls:1.9.2-fpm_2024-06-13_09:13:35.json yourls:1.9.2-fpm] execution failed: exit status 1
Output: 2024-06-13T09:13:37Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:13:37Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "yourls:1.9.2-fpm" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (yourls:1.9.2-fpm): Error response from daemon: No such image: yourls:1.9.2-fpm
	* containerd error: image not found in containerd store: yourls:1.9.2-fpm
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/yourls/manifests/1.9.2-fpm: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_yourls:1.9.2-fpm_2024-06-13_09:13:35.json yourls:1.9.2-fpm] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image yourls:1.9.2-fpm: failed to pull image yourls:1.9.2-fpm: GET https://index.docker.io/v2/library/yourls/manifests/1.9.2-fpm: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
yourls:1.9.2-apache ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_yourls:1.9.2-apache_2024-06-13_09:14:08.json yourls:1.9.2-apache] execution failed: exit status 1
Output: 2024-06-13T09:14:09Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:14:09Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "yourls:1.9.2-apache" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (yourls:1.9.2-apache): Error response from daemon: No such image: yourls:1.9.2-apache
	* containerd error: image not found in containerd store: yourls:1.9.2-apache
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/yourls/manifests/1.9.2-apache: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_yourls:1.9.2-apache_2024-06-13_09:14:08.json yourls:1.9.2-apache] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image yourls:1.9.2-apache: failed to pull image yourls:1.9.2-apache: GET https://index.docker.io/v2/library/yourls/manifests/1.9.2-apache: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
yourls:1.9.2 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_yourls:1.9.2_2024-06-13_09:14:42.json yourls:1.9.2] execution failed: exit status 1
Output: 2024-06-13T09:14:43Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:14:43Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "yourls:1.9.2" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (yourls:1.9.2): Error response from daemon: No such image: yourls:1.9.2
	* containerd error: image not found in containerd store: yourls:1.9.2
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/yourls/manifests/1.9.2: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_yourls:1.9.2_2024-06-13_09:14:42.json yourls:1.9.2] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image yourls:1.9.2: failed to pull image yourls:1.9.2: GET https://index.docker.io/v2/library/yourls/manifests/1.9.2: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
alt:sisyphus ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_alt:sisyphus_2024-06-13_09:15:17.json alt:sisyphus] execution failed: exit status 1
Output: 2024-06-13T09:15:18Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:15:19Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "alt:sisyphus" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (alt:sisyphus): Error response from daemon: No such image: alt:sisyphus
	* containerd error: image not found in containerd store: alt:sisyphus
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/alt/manifests/sisyphus: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_alt:sisyphus_2024-06-13_09:15:17.json alt:sisyphus] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image alt:sisyphus: failed to pull image alt:sisyphus: GET https://index.docker.io/v2/library/alt/manifests/sisyphus: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
alt:p10 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_alt:p10_2024-06-13_09:15:33.json alt:p10] execution failed: exit status 1
Output: 2024-06-13T09:15:34Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:15:35Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "alt:p10" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (alt:p10): Error response from daemon: No such image: alt:p10
	* containerd error: image not found in containerd store: alt:p10
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/alt/manifests/p10: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_alt:p10_2024-06-13_09:15:33.json alt:p10] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image alt:p10: failed to pull image alt:p10: GET https://index.docker.io/v2/library/alt/manifests/p10: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
alt:p9 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_alt:p9_2024-06-13_09:15:50.json alt:p9] execution failed: exit status 1
Output: 2024-06-13T09:15:51Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:15:52Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "alt:p9" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (alt:p9): Error response from daemon: No such image: alt:p9
	* containerd error: image not found in containerd store: alt:p9
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/alt/manifests/p9: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_alt:p9_2024-06-13_09:15:50.json alt:p9] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image alt:p9: failed to pull image alt:p9: GET https://index.docker.io/v2/library/alt/manifests/p9: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
alt:p8 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_alt:p8_2024-06-13_09:16:07.json alt:p8] execution failed: exit status 1
Output: 2024-06-13T09:16:08Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:16:09Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "alt:p8" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (alt:p8): Error response from daemon: No such image: alt:p8
	* containerd error: image not found in containerd store: alt:p8
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/alt/manifests/p8: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_alt:p8_2024-06-13_09:16:07.json alt:p8] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image alt:p8: failed to pull image alt:p8: GET https://index.docker.io/v2/library/alt/manifests/p8: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
amazoncorretto:8u412-al2023-jre ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_amazoncorretto:8u412-al2023-jre_2024-06-13_09:16:22.json amazoncorretto:8u412-al2023-jre] execution failed: exit status 1
Output: 2024-06-13T09:16:23Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:16:24Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "amazoncorretto:8u412-al2023-jre" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (amazoncorretto:8u412-al2023-jre): Error response from daemon: No such image: amazoncorretto:8u412-al2023-jre
	* containerd error: image not found in containerd store: amazoncorretto:8u412-al2023-jre
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/amazoncorretto/manifests/8u412-al2023-jre: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_amazoncorretto:8u412-al2023-jre_2024-06-13_09:16:22.json amazoncorretto:8u412-al2023-jre] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image amazoncorretto:8u412-al2023-jre: failed to pull image amazoncorretto:8u412-al2023-jre: GET https://index.docker.io/v2/library/amazoncorretto/manifests/8u412-al2023-jre: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
amazoncorretto:8u412-al2023 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_amazoncorretto:8u412-al2023_2024-06-13_09:17:16.json amazoncorretto:8u412-al2023] execution failed: exit status 1
Output: 2024-06-13T09:17:17Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:17:17Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "amazoncorretto:8u412-al2023" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (amazoncorretto:8u412-al2023): Error response from daemon: No such image: amazoncorretto:8u412-al2023
	* containerd error: image not found in containerd store: amazoncorretto:8u412-al2023
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/amazoncorretto/manifests/8u412-al2023: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_amazoncorretto:8u412-al2023_2024-06-13_09:17:16.json amazoncorretto:8u412-al2023] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image amazoncorretto:8u412-al2023: failed to pull image amazoncorretto:8u412-al2023: GET https://index.docker.io/v2/library/amazoncorretto/manifests/8u412-al2023: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
amazoncorretto:8u412-al2-native-jre ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_amazoncorretto:8u412-al2-native-jre_2024-06-13_09:18:04.json amazoncorretto:8u412-al2-native-jre] execution failed: exit status 1
Output: 2024-06-13T09:18:05Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:18:06Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "amazoncorretto:8u412-al2-native-jre" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (amazoncorretto:8u412-al2-native-jre): Error response from daemon: No such image: amazoncorretto:8u412-al2-native-jre
	* containerd error: image not found in containerd store: amazoncorretto:8u412-al2-native-jre
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/amazoncorretto/manifests/8u412-al2-native-jre: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_amazoncorretto:8u412-al2-native-jre_2024-06-13_09:18:04.json amazoncorretto:8u412-al2-native-jre] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image amazoncorretto:8u412-al2-native-jre: failed to pull image amazoncorretto:8u412-al2-native-jre: GET https://index.docker.io/v2/library/amazoncorretto/manifests/8u412-al2-native-jre: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
amazoncorretto:8u412-al2-native-jdk ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_amazoncorretto:8u412-al2-native-jdk_2024-06-13_09:18:36.json amazoncorretto:8u412-al2-native-jdk] execution failed: exit status 1
Output: 2024-06-13T09:18:37Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:18:38Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "amazoncorretto:8u412-al2-native-jdk" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (amazoncorretto:8u412-al2-native-jdk): Error response from daemon: No such image: amazoncorretto:8u412-al2-native-jdk
	* containerd error: image not found in containerd store: amazoncorretto:8u412-al2-native-jdk
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/amazoncorretto/manifests/8u412-al2-native-jdk: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_amazoncorretto:8u412-al2-native-jdk_2024-06-13_09:18:36.json amazoncorretto:8u412-al2-native-jdk] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image amazoncorretto:8u412-al2-native-jdk: failed to pull image amazoncorretto:8u412-al2-native-jdk: GET https://index.docker.io/v2/library/amazoncorretto/manifests/8u412-al2-native-jdk: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
amazoncorretto:8u412-al2-generic ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_amazoncorretto:8u412-al2-generic_2024-06-13_09:19:15.json amazoncorretto:8u412-al2-generic] execution failed: exit status 1
Output: 2024-06-13T09:19:16Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:19:17Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "amazoncorretto:8u412-al2-generic" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (amazoncorretto:8u412-al2-generic): Error response from daemon: No such image: amazoncorretto:8u412-al2-generic
	* containerd error: image not found in containerd store: amazoncorretto:8u412-al2-generic
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/amazoncorretto/manifests/8u412-al2-generic: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_amazoncorretto:8u412-al2-generic_2024-06-13_09:19:15.json amazoncorretto:8u412-al2-generic] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image amazoncorretto:8u412-al2-generic: failed to pull image amazoncorretto:8u412-al2-generic: GET https://index.docker.io/v2/library/amazoncorretto/manifests/8u412-al2-generic: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
postfixadmin:fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_postfixadmin:fpm_2024-06-13_09:19:42.json postfixadmin:fpm] execution failed: exit status 1
Output: 2024-06-13T09:19:43Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:19:44Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "postfixadmin:fpm" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (postfixadmin:fpm): Error response from daemon: No such image: postfixadmin:fpm
	* containerd error: image not found in containerd store: postfixadmin:fpm
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/postfixadmin/manifests/fpm: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_postfixadmin:fpm_2024-06-13_09:19:42.json postfixadmin:fpm] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image postfixadmin:fpm: failed to pull image postfixadmin:fpm: GET https://index.docker.io/v2/library/postfixadmin/manifests/fpm: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
postfixadmin:apache ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_postfixadmin:apache_2024-06-13_09:20:18.json postfixadmin:apache] execution failed: exit status 1
Output: 2024-06-13T09:20:19Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:20:20Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "postfixadmin:apache" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (postfixadmin:apache): Error response from daemon: No such image: postfixadmin:apache
	* containerd error: image not found in containerd store: postfixadmin:apache
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/postfixadmin/manifests/apache: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_postfixadmin:apache_2024-06-13_09:20:18.json postfixadmin:apache] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image postfixadmin:apache: failed to pull image postfixadmin:apache: GET https://index.docker.io/v2/library/postfixadmin/manifests/apache: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
postfixadmin:3.3.13-fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_postfixadmin:3.3.13-fpm_2024-06-13_09:20:52.json postfixadmin:3.3.13-fpm] execution failed: exit status 1
Output: 2024-06-13T09:20:53Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:20:53Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "postfixadmin:3.3.13-fpm" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (postfixadmin:3.3.13-fpm): Error response from daemon: No such image: postfixadmin:3.3.13-fpm
	* containerd error: image not found in containerd store: postfixadmin:3.3.13-fpm
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/postfixadmin/manifests/3.3.13-fpm: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_postfixadmin:3.3.13-fpm_2024-06-13_09:20:52.json postfixadmin:3.3.13-fpm] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image postfixadmin:3.3.13-fpm: failed to pull image postfixadmin:3.3.13-fpm: GET https://index.docker.io/v2/library/postfixadmin/manifests/3.3.13-fpm: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
postfixadmin:3.3.13-apache ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_postfixadmin:3.3.13-apache_2024-06-13_09:21:21.json postfixadmin:3.3.13-apache] execution failed: exit status 1
Output: 2024-06-13T09:21:21Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:21:22Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "postfixadmin:3.3.13-apache" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (postfixadmin:3.3.13-apache): Error response from daemon: No such image: postfixadmin:3.3.13-apache
	* containerd error: image not found in containerd store: postfixadmin:3.3.13-apache
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/postfixadmin/manifests/3.3.13-apache: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_postfixadmin:3.3.13-apache_2024-06-13_09:21:21.json postfixadmin:3.3.13-apache] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image postfixadmin:3.3.13-apache: failed to pull image postfixadmin:3.3.13-apache: GET https://index.docker.io/v2/library/postfixadmin/manifests/3.3.13-apache: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
postfixadmin:3.3.13 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_postfixadmin:3.3.13_2024-06-13_09:21:52.json postfixadmin:3.3.13] execution failed: exit status 1
Output: 2024-06-13T09:21:53Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:21:53Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "postfixadmin:3.3.13" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (postfixadmin:3.3.13): Error response from daemon: No such image: postfixadmin:3.3.13
	* containerd error: image not found in containerd store: postfixadmin:3.3.13
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/postfixadmin/manifests/3.3.13: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_postfixadmin:3.3.13_2024-06-13_09:21:52.json postfixadmin:3.3.13] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image postfixadmin:3.3.13: failed to pull image postfixadmin:3.3.13: GET https://index.docker.io/v2/library/postfixadmin/manifests/3.3.13: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
fluentd:v1.16.2-debian-1.1 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_fluentd:v1.16.2-debian-1.1_2024-06-13_09:22:31.json fluentd:v1.16.2-debian-1.1] execution failed: exit status 1
Output: 2024-06-13T09:22:32Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:22:32Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "fluentd:v1.16.2-debian-1.1" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (fluentd:v1.16.2-debian-1.1): Error response from daemon: No such image: fluentd:v1.16.2-debian-1.1
	* containerd error: image not found in containerd store: fluentd:v1.16.2-debian-1.1
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/fluentd/manifests/v1.16.2-debian-1.1: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_fluentd:v1.16.2-debian-1.1_2024-06-13_09:22:31.json fluentd:v1.16.2-debian-1.1] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image fluentd:v1.16.2-debian-1.1: failed to pull image fluentd:v1.16.2-debian-1.1: GET https://index.docker.io/v2/library/fluentd/manifests/v1.16.2-debian-1.1: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
fluentd:v1.16-debian-1 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_fluentd:v1.16-debian-1_2024-06-13_09:22:58.json fluentd:v1.16-debian-1] execution failed: exit status 1
Output: 2024-06-13T09:22:59Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:22:59Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "fluentd:v1.16-debian-1" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (fluentd:v1.16-debian-1): Error response from daemon: No such image: fluentd:v1.16-debian-1
	* containerd error: image not found in containerd store: fluentd:v1.16-debian-1
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/fluentd/manifests/v1.16-debian-1: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_fluentd:v1.16-debian-1_2024-06-13_09:22:58.json fluentd:v1.16-debian-1] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image fluentd:v1.16-debian-1: failed to pull image fluentd:v1.16-debian-1: GET https://index.docker.io/v2/library/fluentd/manifests/v1.16-debian-1: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
fluentd:v1.16.2-1.1 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_fluentd:v1.16.2-1.1_2024-06-13_09:23:25.json fluentd:v1.16.2-1.1] execution failed: exit status 1
Output: 2024-06-13T09:23:26Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:23:27Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "fluentd:v1.16.2-1.1" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (fluentd:v1.16.2-1.1): Error response from daemon: No such image: fluentd:v1.16.2-1.1
	* containerd error: image not found in containerd store: fluentd:v1.16.2-1.1
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/fluentd/manifests/v1.16.2-1.1: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_fluentd:v1.16.2-1.1_2024-06-13_09:23:25.json fluentd:v1.16.2-1.1] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image fluentd:v1.16.2-1.1: failed to pull image fluentd:v1.16.2-1.1: GET https://index.docker.io/v2/library/fluentd/manifests/v1.16.2-1.1: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
fluentd:v1.16-1 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_fluentd:v1.16-1_2024-06-13_09:23:43.json fluentd:v1.16-1] execution failed: exit status 1
Output: 2024-06-13T09:23:44Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:23:44Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "fluentd:v1.16-1" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (fluentd:v1.16-1): Error response from daemon: No such image: fluentd:v1.16-1
	* containerd error: image not found in containerd store: fluentd:v1.16-1
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/fluentd/manifests/v1.16-1: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_fluentd:v1.16-1_2024-06-13_09:23:43.json fluentd:v1.16-1] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image fluentd:v1.16-1: failed to pull image fluentd:v1.16-1: GET https://index.docker.io/v2/library/fluentd/manifests/v1.16-1: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
fluentd:v1.16.0-debian-1.0 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_fluentd:v1.16.0-debian-1.0_2024-06-13_09:24:00.json fluentd:v1.16.0-debian-1.0] execution failed: exit status 1
Output: 2024-06-13T09:24:01Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:24:02Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "fluentd:v1.16.0-debian-1.0" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (fluentd:v1.16.0-debian-1.0): Error response from daemon: No such image: fluentd:v1.16.0-debian-1.0
	* containerd error: image not found in containerd store: fluentd:v1.16.0-debian-1.0
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/fluentd/manifests/v1.16.0-debian-1.0: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_fluentd:v1.16.0-debian-1.0_2024-06-13_09:24:00.json fluentd:v1.16.0-debian-1.0] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image fluentd:v1.16.0-debian-1.0: failed to pull image fluentd:v1.16.0-debian-1.0: GET https://index.docker.io/v2/library/fluentd/manifests/v1.16.0-debian-1.0: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
sapmachine:ubuntu-noble ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_sapmachine:ubuntu-noble_2024-06-13_09:24:22.json sapmachine:ubuntu-noble] execution failed: exit status 1
Output: 2024-06-13T09:24:23Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:24:23Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "sapmachine:ubuntu-noble" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (sapmachine:ubuntu-noble): Error response from daemon: No such image: sapmachine:ubuntu-noble
	* containerd error: image not found in containerd store: sapmachine:ubuntu-noble
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/sapmachine/manifests/ubuntu-noble: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_sapmachine:ubuntu-noble_2024-06-13_09:24:22.json sapmachine:ubuntu-noble] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image sapmachine:ubuntu-noble: failed to pull image sapmachine:ubuntu-noble: GET https://index.docker.io/v2/library/sapmachine/manifests/ubuntu-noble: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
sapmachine:ubuntu-jammy ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_sapmachine:ubuntu-jammy_2024-06-13_09:24:52.json sapmachine:ubuntu-jammy] execution failed: exit status 1
Output: 2024-06-13T09:24:53Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:24:54Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "sapmachine:ubuntu-jammy" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (sapmachine:ubuntu-jammy): Error response from daemon: No such image: sapmachine:ubuntu-jammy
	* containerd error: image not found in containerd store: sapmachine:ubuntu-jammy
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/sapmachine/manifests/ubuntu-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_sapmachine:ubuntu-jammy_2024-06-13_09:24:52.json sapmachine:ubuntu-jammy] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image sapmachine:ubuntu-jammy: failed to pull image sapmachine:ubuntu-jammy: GET https://index.docker.io/v2/library/sapmachine/manifests/ubuntu-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
sapmachine:ubuntu-focal ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_sapmachine:ubuntu-focal_2024-06-13_09:25:27.json sapmachine:ubuntu-focal] execution failed: exit status 1
Output: 2024-06-13T09:25:28Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:25:29Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "sapmachine:ubuntu-focal" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (sapmachine:ubuntu-focal): Error response from daemon: No such image: sapmachine:ubuntu-focal
	* containerd error: image not found in containerd store: sapmachine:ubuntu-focal
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/sapmachine/manifests/ubuntu-focal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_sapmachine:ubuntu-focal_2024-06-13_09:25:27.json sapmachine:ubuntu-focal] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image sapmachine:ubuntu-focal: failed to pull image sapmachine:ubuntu-focal: GET https://index.docker.io/v2/library/sapmachine/manifests/ubuntu-focal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
sapmachine:ubuntu-24.04 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_sapmachine:ubuntu-24.04_2024-06-13_09:25:58.json sapmachine:ubuntu-24.04] execution failed: exit status 1
Output: 2024-06-13T09:25:59Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:26:00Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "sapmachine:ubuntu-24.04" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (sapmachine:ubuntu-24.04): Error response from daemon: No such image: sapmachine:ubuntu-24.04
	* containerd error: image not found in containerd store: sapmachine:ubuntu-24.04
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/sapmachine/manifests/ubuntu-24.04: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_sapmachine:ubuntu-24.04_2024-06-13_09:25:58.json sapmachine:ubuntu-24.04] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image sapmachine:ubuntu-24.04: failed to pull image sapmachine:ubuntu-24.04: GET https://index.docker.io/v2/library/sapmachine/manifests/ubuntu-24.04: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
sapmachine:ubuntu-22.04 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_sapmachine:ubuntu-22.04_2024-06-13_09:26:23.json sapmachine:ubuntu-22.04] execution failed: exit status 1
Output: 2024-06-13T09:26:25Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:26:25Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "sapmachine:ubuntu-22.04" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (sapmachine:ubuntu-22.04): Error response from daemon: No such image: sapmachine:ubuntu-22.04
	* containerd error: image not found in containerd store: sapmachine:ubuntu-22.04
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/sapmachine/manifests/ubuntu-22.04: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_sapmachine:ubuntu-22.04_2024-06-13_09:26:23.json sapmachine:ubuntu-22.04] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image sapmachine:ubuntu-22.04: failed to pull image sapmachine:ubuntu-22.04: GET https://index.docker.io/v2/library/sapmachine/manifests/ubuntu-22.04: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
varnish:stable ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_varnish:stable_2024-06-13_09:26:49.json varnish:stable] execution failed: exit status 1
Output: 2024-06-13T09:26:50Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:26:51Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "varnish:stable" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (varnish:stable): Error response from daemon: No such image: varnish:stable
	* containerd error: image not found in containerd store: varnish:stable
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/varnish/manifests/stable: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_varnish:stable_2024-06-13_09:26:49.json varnish:stable] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image varnish:stable: failed to pull image varnish:stable: GET https://index.docker.io/v2/library/varnish/manifests/stable: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
varnish:old ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new9/Trivy_scan_varnish:old_2024-06-13_09:27:13.json varnish:old] execution failed: exit status 1
Output: 2024-06-13T09:27:14Z	INFO	Vulnerability scanning is enabled
2024-06-13T09:27:15Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "varnish:old" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (varnish:old): Error response from daemon: No such image: varnish:old
	* containerd error: image not found in containerd store: varnish:old
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/varnish/manifests/old: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new9/DockerScout_scan_varnish:old_2024-06-13_09:27:13.json varnish:old] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image varnish:old: failed to pull image varnish:old: GET https://index.docker.io/v2/library/varnish/manifests/old: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> Command [/usr/local/bin/grype -o sarif --file /root/docker-vulnerability-analyzer/out/official/new9/Grype_scan_varnish:old_2024-06-13_09:27:13.json varnish:old] execution failed: exit status 1
Output: [31mfailed to catalog: errors occurred attempting to resolve 'varnish:old':
  - docker: unable to inspect existing image: Error response from daemon: No such image: varnish:old
  - podman: podman not available: no host address
  - containerd: pull failed: failed to copy: httpReadSeeker: failed open: unexpected status code https://registry-1.docker.io/v2/library/varnish/manifests/sha256:b8370ea0069c03b3b14ae335d01ac6fe9c1a2416cb58a04374c7a05f794f5696: 429 Too Many Requests - Server message: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  - oci-registry: failed to get image descriptor from registry: GET https://index.docker.io/v2/library/varnish/manifests/old: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  - additionally, the following providers failed with file does not exist: docker-archive, oci-archive, oci-dir, singularity, oci-dir, local-file, local-directory[0m
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> Command [/usr/local/bin/snyk container test varnish:old --sarif-file-output=/root/docker-vulnerability-analyzer/out/official/new9/Snyk_scan_varnish:old_2024-06-13_09:27:13.json] execution failed: exit status 2
Output: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit

varnish:fresh ----------- Error pulling image varnish:fresh: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
varnish:7.5.0 ----------- Error pulling image varnish:7.5.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
varnish:7.5 ----------- Error pulling image varnish:7.5: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
archlinux:multilib-devel-20240101.0.204074 ----------- Error pulling image archlinux:multilib-devel-20240101.0.204074: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
archlinux:multilib-devel ----------- Error pulling image archlinux:multilib-devel: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
archlinux:base-devel-20240101.0.204074 ----------- Error pulling image archlinux:base-devel-20240101.0.204074: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
archlinux:base-devel ----------- Error pulling image archlinux:base-devel: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
archlinux:base-20240101.0.204074 ----------- Error pulling image archlinux:base-20240101.0.204074: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
friendica:stable-fpm ----------- Error pulling image friendica:stable-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
friendica:stable-apache ----------- Error pulling image friendica:stable-apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
friendica:stable ----------- Error pulling image friendica:stable: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
friendica:fpm ----------- Error pulling image friendica:fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
friendica:dev-fpm ----------- Error pulling image friendica:dev-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
caddy:builder-alpine ----------- Error pulling image caddy:builder-alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
caddy:builder ----------- Error pulling image caddy:builder: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
caddy:alpine ----------- Error pulling image caddy:alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
caddy:2.8.4-builder-alpine ----------- Error pulling image caddy:2.8.4-builder-alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
caddy:2.8.4-builder ----------- Error pulling image caddy:2.8.4-builder: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
monica:5.0.0-beta.4-fpm ----------- Error pulling image monica:5.0.0-beta.4-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
monica:5.0.0-beta.4-apache ----------- Error pulling image monica:5.0.0-beta.4-apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
monica:5.0.0-beta-fpm ----------- Error pulling image monica:5.0.0-beta-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
monica:5.0.0-beta-apache ----------- Error pulling image monica:5.0.0-beta-apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
monica:5.0-fpm ----------- Error pulling image monica:5.0-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
phpmyadmin:fpm ----------- Error pulling image phpmyadmin:fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
phpmyadmin:apache ----------- Error pulling image phpmyadmin:apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
phpmyadmin:5.2.1-fpm ----------- Error pulling image phpmyadmin:5.2.1-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
phpmyadmin:5.2.1-apache ----------- Error pulling image phpmyadmin:5.2.1-apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
phpmyadmin:5.2.1 ----------- Error pulling image phpmyadmin:5.2.1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
hitch:1.8.0-1 ----------- Error pulling image hitch:1.8.0-1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
hitch:1.8.0 ----------- Error pulling image hitch:1.8.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
hitch:1.8 ----------- Error pulling image hitch:1.8: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
hitch:1 ----------- Error pulling image hitch:1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
hitch:1.7.3-1 ----------- Error pulling image hitch:1.7.3-1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
almalinux:minimal ----------- Error pulling image almalinux:minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
almalinux:9.4-minimal-20240530 ----------- Error pulling image almalinux:9.4-minimal-20240530: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
almalinux:9.4-minimal ----------- Error pulling image almalinux:9.4-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
almalinux:9.4-20240530 ----------- Error pulling image almalinux:9.4-20240530: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
almalinux:9.4 ----------- Error pulling image almalinux:9.4: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
dart:stable-sdk ----------- Error pulling image dart:stable-sdk: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
dart:stable ----------- Error pulling image dart:stable: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
dart:sdk ----------- Error pulling image dart:sdk: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
dart:beta-sdk ----------- Error pulling image dart:beta-sdk: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
dart:beta ----------- Error pulling image dart:beta: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
eclipse-temurin:8u412-b08-jre-ubi9-minimal ----------- Error pulling image eclipse-temurin:8u412-b08-jre-ubi9-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
eclipse-temurin:8u412-b08-jdk-ubi9-minimal ----------- Error pulling image eclipse-temurin:8u412-b08-jdk-ubi9-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
eclipse-temurin:8-ubi9-minimal ----------- Error pulling image eclipse-temurin:8-ubi9-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
eclipse-temurin:8-jre-ubi9-minimal ----------- Error pulling image eclipse-temurin:8-jre-ubi9-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
eclipse-temurin:8-jdk-ubi9-minimal ----------- Error pulling image eclipse-temurin:8-jdk-ubi9-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
ibm-semeru-runtimes:open-8u412-b08-jre-jammy ----------- Error pulling image ibm-semeru-runtimes:open-8u412-b08-jre-jammy: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
ibm-semeru-runtimes:open-8u412-b08-jre-focal ----------- Error pulling image ibm-semeru-runtimes:open-8u412-b08-jre-focal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
ibm-semeru-runtimes:open-8u412-b08-jre ----------- Error pulling image ibm-semeru-runtimes:open-8u412-b08-jre: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
ibm-semeru-runtimes:open-8u412-b08-jdk-jammy ----------- Error pulling image ibm-semeru-runtimes:open-8u412-b08-jdk-jammy: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
ibm-semeru-runtimes:open-8u412-b08-jdk-focal ----------- Error pulling image ibm-semeru-runtimes:open-8u412-b08-jdk-focal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rockylinux:9.3.20231119-minimal ----------- Error pulling image rockylinux:9.3.20231119-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rockylinux:9.3.20231119 ----------- Error pulling image rockylinux:9.3.20231119: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rockylinux:9.3-minimal ----------- Error pulling image rockylinux:9.3-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rockylinux:9.3 ----------- Error pulling image rockylinux:9.3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rockylinux:9-minimal ----------- Error pulling image rockylinux:9-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
api-firewall:0.7.3 ----------- Error pulling image api-firewall:0.7.3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
api-firewall:0.7.2 ----------- Error pulling image api-firewall:0.7.2: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
api-firewall:0.7.1 ----------- Error pulling image api-firewall:0.7.1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
api-firewall:0.7.0 ----------- Error pulling image api-firewall:0.7.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
api-firewall:0.6.17 ----------- Error pulling image api-firewall:0.6.17: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5.7.0 ----------- Error pulling image emqx:5.7.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5.7 ----------- Error pulling image emqx:5.7: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5 ----------- Error pulling image emqx:5: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5.6.1 ----------- Error pulling image emqx:5.6.1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5.6 ----------- Error pulling image emqx:5.6: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
satosa:bookworm ----------- Error pulling image satosa:bookworm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
satosa:8.4.0-bookworm ----------- Error pulling image satosa:8.4.0-bookworm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
satosa:8.4.0 ----------- Error pulling image satosa:8.4.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
satosa:8.4-bookworm ----------- Error pulling image satosa:8.4-bookworm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
satosa:8.4 ----------- Error pulling image satosa:8.4: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
unit:php8.3 ----------- Error pulling image unit:php8.3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
unit:php8.2 ----------- Error pulling image unit:php8.2: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
unit:php8 ----------- Error pulling image unit:php8: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
unit:php ----------- Error pulling image unit:php: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
unit:1.32.1-php8.3 ----------- Error pulling image unit:1.32.1-php8.3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:scala ----------- Error pulling image spark:scala: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:r ----------- Error pulling image spark:r: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:python3-java17 ----------- Error pulling image spark:python3-java17: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:python3 ----------- Error pulling image spark:python3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:3.5.1-scala2.12-java17-ubuntu ----------- Error pulling image spark:3.5.1-scala2.12-java17-ubuntu: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:4.28.0 ----------- Error pulling image liquibase:4.28.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:4.28 ----------- Error pulling image liquibase:4.28: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:alpine ----------- Error pulling image liquibase:alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:4.28.0-alpine ----------- Error pulling image liquibase:4.28.0-alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:4.28-alpine ----------- Error pulling image liquibase:4.28-alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
